client/my-sites/themes/single-site-jetpack.jsx

```diff
@@ -79,7 +81,9 @@ const ConnectedSingleSiteJetpack = connectOptions(
 			<div>
 				<SidebarNavigation />
 				<CurrentTheme siteId={ siteId } />
-				<ThemeShowcase { ...props } siteId={ siteId }>
+				<ThemeShowcase { ...props }
+					siteId={ siteId }
+					emptyContent={ <div /> } >
```

Comment:
Does this mean that in production, where we don't show the lower list, an empty search would not result in the "No Themes Found" message?